en/gps_compass/rtk_gps_locosys_r1.md

```diff
@@ -0,0 +1,100 @@
+# Locosys RTK GPS
+
+The [LOCOSYS Hawk R1](https://www.locosystech.com/en/product/hawk-r1.html) is a dual-frequency [RTK GPS module](../gps_compass/rtk_gps.md) receiver designed based on compatibility with Pixhawk from Locosys.
+The module can act as rover when installed on the aircraft. The receiver is capable of concurrently tracking all global civil navigation systems, including GPS, GLONASS, GALILEO, BEIDOU and QZSS. It acquires both L1 and L5 signals at the same time while providing the centimeter-level RTK positioning accuracy.
+
+The built-in lightweight helical antenna not only enhances RTK positioning stability, but also increases the flight time of the drone. The fast Time-To-First-Fix, RTK convergence, superior sensitivity, low power consumption make it a better choice for Pixhawk-based platform UAVs.

```

Comment:
> but also increases the flight time of the drone. 

How? That sound unlikely.

Reply:
OK, I assume this is because of the low power consumption.

```suggestion
The built-in lightweight helical antenna enhances RTK positioning stability.
The fast time-to-first-fix, RTK convergence, superior sensitivity, low power consumption make it a better choice for Pixhawk-based platform UAVs.

```